core/src/apps/eos/get_public_key.py

```diff
@@ -24,7 +24,8 @@ async def get_public_key(msg: EosGetPublicKey, keychain: Keychain) -> EosPublicK
 
     public_key = secp256k1.publickey(node.private_key(), True)
     wif = public_key_to_wif(public_key)
+    path = paths.address_n_to_str(msg.address_n)
```

Comment:
Can calculate the `path` inside `if msg.show_display` - it is not used otherwise

Reply:
e49b023